pyperclip/__init__.py

```diff
@@ -79,6 +79,8 @@ def determine_clipboard():
             return init_xsel_clipboard()
         if _executable_exists("klipper") and _executable_exists("qdbus"):
             return init_klipper_clipboard()
+        if _executable_exists("termux-clipboard-get") and _executable_exists("termux-clipboard-set"):
```

Comment:
`HAS_DISPLAY` is `False` on Android. This should be as an `elif` to the `HAS_DISPLAY` check.